tests/libp2p/pubsub/component/test_gossipsub_message_handling.nim

```diff
@@ -139,7 +147,10 @@ suite "GossipSub Component - Message Handling":
       isHighPriority = false,
     )
 
-    checkUntilTimeout:
```

Comment:
`checkUntilTimeout` must not be used when asserting that messages are not received as first check will make it satisfactory.